orb/src/jobs/heroku-provision.yml

```diff
@@ -2,11 +2,26 @@ parameters:
   executor:
     default: default
     type: executor
+  aws-account-id:
+    default: ''
+    type: string
+  system-code:
+    default: ''
+    type: string
 
 executor: << parameters.executor >>
 
 steps:
   - attach-workspace
+  - when:
```

Comment:
**praise:** great commit message description of why you chose this solution. The code is tidy and readable so it def feels to me like the best approach 🎉